pkg/openservicebroker/types.go

```diff
@@ -0,0 +1,157 @@
+package openservicebroker
```

Comment:
Put these in their own package - types and implementations have nothing to do with each other.  I'd suggest `pkg/openservicebroker/api`